### PR TITLE
continue to fix for membership in the set

### PR DESCRIPTION
**Pull request by dfenglei**
The former commit can fix the type error, but it still has some error in checking for the membersip in set. 
So I want to continue to fix this issue #101 .

### CODE REVIEW

File: drqa/tokenizers/spacy_tokenizer.py [hunk 1]
```diff
@@ -25,19 +25,19 @@ def __init__(self, **kwargs):
         model = kwargs.get('model', 'en')
         self.annotators = copy.deepcopy(kwargs.get('annotators', set()))
         nlp_kwargs = {'parser': False}
-        if not {'lemma', 'pos', 'ner'} in self.annotators:
+        if ('lemma' not in self.annotators) and ('pos' not in self.annotators) and ('ner' not in self.annotators):
```
Comment (ajfisch) [inline drqa/tokenizers/spacy_tokenizer.py]: `if not any([p in self.annotators for p in ['lemma', 'pos', 'ner']):`

File: drqa/tokenizers/spacy_tokenizer.py [hunk 2]
```diff
@@ -25,19 +25,19 @@ def __init__(self, **kwargs):
...
+        if 'ner' not in self.annotators:
             nlp_kwargs['entity'] = False
         self.nlp = spacy.load(model, **nlp_kwargs)
 
     def tokenize(self, text):
         # We don't treat new lines as tokens.
         clean_text = text.replace('\n', ' ')
         tokens = self.nlp.tokenizer(clean_text)
-        if {'lemma', 'pos', 'ner'} in self.annotators:
+        if ('lemma' in self.annotators) or ('pos' in self.annotators) or('ner' in self.annotators) :
```
Comment (ajfisch) [inline drqa/tokenizers/spacy_tokenizer.py]: `if any([p in self.annotators for p in ['lemma', 'pos', 'ner']):`

File: drqa/tokenizers/spacy_tokenizer.py (hunk 1 above)
Comment (ajfisch) [reply drqa/tokenizers/spacy_tokenizer.py]: (or `if len({'lemma', 'pos', 'ner'} & self.annotators) == 0`)

Comment (dfenglei) [reply drqa/tokenizers/spacy_tokenizer.py]: More simple and graceful.

**Comment by dfenglei** [comment]
I will update the code later.

**Comment by ajfisch** [comment]
thanks!

**Pull request merged**